.github/workflows/lint-test.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
         uses: actions/checkout@v1
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.0.0-alpha.2
+        uses: helm/kind-action@master
         with:
           installLocalPathProvisioner: true
```

Comment:
```suggestion
          install_local_path_provisioner: true
```

Reply:
Ah too late, thank you yes I completely forgot to update the input key after https://github.com/helm/kind-action/pull/10 💯